packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -253,7 +255,7 @@ export class LitNodeClientNodeJs
       throw new Error('Failed to verify capabilities for resource');
     }
 
-    const nonce = this.getLatestBlockhash();
+    let nonce = await this.getLatestBlockhash();
```

Comment:
We're not mutating `nonce`; it should be a `const` (this is also an eslint error for `prefer-const`)